src/Service.Host/client/src/components/importBooks/tabs/ImpBookTab.js

```diff
@@ -188,18 +133,42 @@ function ImpBookTab({
                         onChange={handleFieldChange}
                         type="number"
                         disabled={!allowedToEdit}
+                        data-testid="createdBy"
```

Comment:
out of interst: did getByLabelText not work for this Dropdown?

Reply:
I think I forgot to take this out, couldn't get this to work either. I can't remember if I had required on the dropdown as I'm sure that broke it, I'll have a quick play about and see if I can test it

Reply:
Yeah, getByLabelText doesn't work for it